cmd/query/app/http_handler.go

```diff
@@ -158,6 +161,39 @@ func (aH *APIHandler) formatRoute(route string, args ...interface{}) string {
 	return fmt.Sprintf("/%s"+route, args...)
 }
 
+func (aH *APIHandler) transformOTLP(w http.ResponseWriter, r *http.Request) {
```

Comment:
please move the function to a place in the file so that it reflects its relative position in RegisterRoutes